### ISSUE TITLE
Random Daily Quote

### ISSUE THREAD

**Issue by keolo** [SilentVoid13/Templater#455]
When I use `tp.web.daily_quote()` and create a new daily note in the future, e.g. to plan tomorrow, it returns today's daily quote.

It would be cool if there was an option to return a random quote for `tp.web.daily_quote()` or to create a new function to do such.

Looks like https://quotes.rest/ doesn't have a free option for random quotes. However, I did find this https://type.fit/api/quotes and there might be other options out there. Maybe could add a JSON file in this repo to query?

**Comment by shabegom** [comment]
@keolo try this command out for a random quote every time:

```
<%*
const response = await fetch(`https://api.quotable.io/random`);
const     qod = await response.json();
const finalQuote = `> ${qod.content}
>
> &mdash; <cite>${qod.author}</cite>✍️`
tR += finalQuote
%>
```

**Comment by shabegom** [comment]
resolved in 1.9.10
